Gruntfile.js

```diff
@@ -36,6 +36,16 @@ module.exports = function(grunt) {
 		dateString: now.substr(0, now.indexOf("T")),
 
 		pkg: grunt.file.readJSON("package.json"),
+		commonDtsFileCreation: grunt.file.write(require("path").join("lib", "common", ".d.ts"), ""),
+		
+		// When there are node_modules inside lib\common directory, CLI behaves incorrectly, so delete this dir.
+		removeCommonNodeModules: require("rimraf")(require("path").join("lib", "common", "node_modules"), function(err) {
```

Comment:
You can either concat the path to a single string, or require path and use it everywhere.